sidebars.js

```diff
@@ -42,6 +42,7 @@ const sidebars = {
       type: 'category',
       label: 'SDKs',
       items: [
+        'workflows/bind-passkey',
```

Comment:
Do we need this in the SDKs section too? Was this just a placeholder for testing before "Workflows" existed?

Reply:
I was thinking we could move Bind, your SDK Setup and Auth into the "SDK section" and remove the older stuff from that section completely.

Reply:
That makes sense to me